### PR TITLE
Add 2.7 Chainstack release notes

### PR DESCRIPTION
**Pull request by SozinM**

### CODE REVIEW

**Review by akegaviar — COMMENTED** [review]
Are we release requests? If we do, we need to include it.

**Review by NikitosnikN — APPROVED** [review]

**Comment by NikitosnikN** [comment]
branch should be rebased after https://github.com/chainstack/docs/pull/392 merge

File: docs/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ meta:
...
+## Chainstack 2.7
+
+*November 30, 2021*
+
+### What's new
+
+* **Protocols**. [Fantom](/blockchains/fantom) support. You can now deploy full and archive Fantom shared nodes on mainnet and testnet.
+* **Protocols**. [Tezos](/platform/join-a-public-network#join-a-tezos-network) protocol: Added dedicated node support.
+* **Documentation**. Fantom [general description](/blockchains/fantom), [operations](/operations/fantom), [ERC-721 contract tutorial](/tutorials/fantom/erc-721-collection-contract-with-truffle-and-openzeppelin).
+* **Node metrics**. Added node [request metrics charts](/platform/view-node-requests-metrics) for public chain protocols.
```
Comment (easeev) [inline docs/release-notes.md:20]: @akegaviar please proofread "request metrics charts"

Comment (akegaviar) [reply docs/release-notes.md]: Done

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**